src/VTK/VtkWellboreTrajectoryRepresentation.cxx

```diff
@@ -43,38 +35,25 @@ VtkWellboreTrajectoryRepresentation::~VtkWellboreTrajectoryRepresentation()
 		epcPackageSubRepresentation = nullptr;
 	}
 
-	if (polyline != nullptr) {
-		delete polyline;
-		polyline = nullptr;
-	}
-	if (head != nullptr) {
-		delete head;
-		head = nullptr;
-	}
-	if (head != nullptr) {
-		delete head;
-		head = nullptr;
-	}
-	if (text != nullptr) {
-		delete text;
-		text = nullptr;
-	}
+	polyline.~VtkWellboreTrajectoryRepresentationPolyLine();

```

Comment:
For the three lines :
No need at all. Only a destructor on a pointer is needed to be called.
This variable here is automatically deleted automatically (that's one of the advantage compared to using pointer)